Basestation_Software.Web/Core/Components/RamanGraph.razor

```diff
@@ -0,0 +1,146 @@
+﻿@inject RoveCommService _RoveCommService
+
+<div class="card full-height">
+   <div class="card-header">
+       <h5>Raman Spectroscopy Graph</h5>
+   </div>
+   <p>
+       <RadzenChart>
+          <RadzenLineSeries Data="@ramanGraph" CategoryProperty="Raman_Shift" ValueProperty="Intensity" />
+       </RadzenChart>
+   </p>
+   <button @onclick=RequestRamanData> Request Raman Spectroscopy Data </button>
+</div> 
+
+
+@code {
+    //Variables
+    private static List<ushort> _raman1 = new List<ushort>(500),
+    _raman2 = new List<ushort>(500),
+    _raman3 = new List<ushort>(500),
+    _raman4 = new List<ushort>(500),
+    _raman5 = new List<ushort>(48);
+
+
+    class DataItem
+    {
+        public int Raman_Shift { get; set; }
+        public double Intensity { get; set; }
+
+        public DataItem()
+        {
+            Raman_Shift = 0;
+            Intensity = 0f;
+        }
+    }
+
+    DataItem[] ramanGraph = new DataItem[2048];
```

Comment:
Do you want this to be public? Consider making it private and naming _ramanGraph.